nautobot/extras/choices.py

```diff
@@ -71,11 +71,11 @@ class CustomFieldTypeChoices(ChoiceSet):
 
 
 #
-# CustomLinks
+# Button Classes
 #
 
 
-class CustomLinkButtonClassChoices(ChoiceSet):
+class ButtonClassChoices(ChoiceSet):
```

Comment:
Since this is going into `develop` let's not assume no one is using this. Please use the `@class_deprecated_in_favor_of` decorator to seamlessly rename this class.

(suggesstion may not pass Black, Flake8)
```suggestion
@class_deprecated_in_favor_of(ButtonClassChoices)
class CustomLinkButtonClassChoices(ButtonClassChoices):
    pass

class ButtonClassChoices(ChoiceSet):
```

Reply:
Added in latest commits.